src/jit/codegenarm64.cpp

```diff
@@ -5453,7 +5452,7 @@ void CodeGen::genJmpMethod(GenTreePtr jmp)
 
         // Is register argument already in the right register?
         // If not load it from its stack location.
-        var_types  loadType  = varDsc->lvaArgType();
+        var_types loadType = compiler->mangleVarArgsType(genActualType(varDsc->TypeGet()));
```

Comment:
As per my understanding structs of size 16 are passed in two registers on Arm64.  Does this code also handle the case of "jmp method" where one of the args of the method is a 16-byte struct passed in two regs?